[2025-03-16T02:36:46.382+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T02:36:46.395+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T01:00:00+00:00 [queued]>
[2025-03-16T02:36:46.400+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T01:00:00+00:00 [queued]>
[2025-03-16T02:36:46.401+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T02:36:46.410+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.check_minio_connection> on 2025-03-16 01:00:00+00:00
[2025-03-16T02:36:46.417+0000] {standard_task_runner.py:72} INFO - Started process 225 to run task
[2025-03-16T02:36:46.421+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.check_minio_connection', 'scheduled__2025-03-16T01:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp1_25p9l9']
[2025-03-16T02:36:46.424+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask bronze_layer_group.check_minio_connection
[2025-03-16T02:36:46.485+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T01:00:00+00:00 [running]> on host 3174348cc1e9
[2025-03-16T02:36:46.562+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T01:00:00+00:00'
[2025-03-16T02:36:46.564+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T02:36:46.571+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T02:36:46.575+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T02:36:46.575+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T02:36:46.576+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T02:36:47.229+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-03-16T02:36:47.250+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T02:36:47.250+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.check_minio_connection, run_id=scheduled__2025-03-16T01:00:00+00:00, execution_date=20250316T010000, start_date=20250316T023646, end_date=20250316T023647
[2025-03-16T02:36:47.295+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T02:36:47.320+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T02:36:47.321+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
